hourglass.py

```diff
@@ -123,6 +134,9 @@ def resume_train(self, batch_size, model_save_base_dir, model_json, model_weight
         print('Loss function selected: {}'.format(loss_str))
         self.loss = get_loss_from_string(loss_str)
 
+        print('Image augmentation strength selected: {}'.format(image_aug_strength))
+        self.img_aug_strength = get_strength_enum_from_string(image_aug_strength)
```

Comment:
I think it would be more consistent with the other params if `image_aug_strength` and `loss` get passed through to `_start_train` in the same way `batch_size` does instead of making instance variables that get initialized at some point after init is called.

Reply:
yep good call. It was getting confusing for me too haha. I'll change that.

Reply:
Fixed